testnet/stacks-node/src/config.rs

```diff
@@ -292,8 +292,16 @@ impl ConfigFile {
             "02afeae522aab5f8c99a00ddf75fbcb4a641e052dd48836408d9cf437344b63516@seed-1.mainnet.stacks.co:20444",
             "03652212ea76be0ed4cd83a25c06e57819993029a7b9999f7d63c36340b34a4e62@seed-2.mainnet.stacks.co:20444"].join(",");
 
+        let height_hint_nodes = [
+            "seed-0.mainnet.stacks.co:20443",
+            "seed-1.mainnet.stacks.co:20443",
+            "seed-2.mainnet.stacks.co:20443",
+        ]
+        .join(",");
```

Comment:
So, I'm not convinced that we need `height_hint_nodes` at all.  This information is already available to you -- the `PeerNetwork` struct already tracks each of its neighbors' highest known burnchain block heights in its `ConversationP2P` structs.  You ought to be able to add a method to `PeerNetwork` that just goes through the set of `ConversationP2P` structs and extracts the highest reported burn block height (if there are any nodes connected at all).

Reply:
But we are trying to get _Stacks_ height, not burnchain block heights. Using the PeerNetwork struct would require instantiating a new network object (and driving the events in it) during startup. That seems way more error prone than just directly querying the `v2/info` endpoint.